lib/cli.rb

```diff
@@ -774,6 +774,11 @@ def self.parse_inspect_command_options(host, options)
       inspected_filters = description.filter_definitions("inspect")
 
       if options[:verbose]
+        if description[:environment].attributes.include?("system_type")
```

Comment:
This check needs to be adapted I think. We always store a `system_attribute`, not only for docker containers. For remote and local systems it's `remote` and `local`, for docker containers `docker`.